### PR TITLE
only mail to infraswat

### PR DESCRIPTION
**Pull request by TyOverby**

### CODE REVIEW

**Comment by TyOverby** [comment]
@tannergooding @mmitche

File: netci.groovy [hunk 1]
```diff
@@ -9,9 +9,9 @@ def project = GithubProject
 static void addEmailPublisher(def myJob) {
   myJob.with {
     publishers {
-      extendedEmail('$DEFAULT_RECIPIENTS, cc:mlinfraswat@microsoft.com', '$DEFAULT_SUBJECT', '$DEFAULT_CONTENT') {
-        trigger('Aborted', '$PROJECT_DEFAULT_SUBJECT', '$PROJECT_DEFAULT_CONTENT', null, true, true, true, true)
-        trigger('Failure', '$PROJECT_DEFAULT_SUBJECT', '$PROJECT_DEFAULT_CONTENT', null, true, true, true, true)
+      extendedEmail('cc:mlinfraswat@microsoft.com', '$DEFAULT_SUBJECT', '$DEFAULT_CONTENT') {
```
Comment (tannergooding) [inline netci.groovy]: Remove `cc:`

File: netci.groovy [hunk 2]
```diff
@@ -9,9 +9,9 @@ def project = GithubProject
...
+        trigger('Aborted', '$PROJECT_DEFAULT_SUBJECT', '$PROJECT_DEFAULT_CONTENT', null, false, false, false, true)
```
Comment (tannergooding) [inline netci.groovy]: Add comment detailing what each parameter is

File: netci.groovy (hunk 1 above)
Comment (TyOverby) [inline netci.groovy]: Done

File: netci.groovy (hunk 2 above)
Comment (TyOverby) [inline netci.groovy]: done

**Comment by tannergooding** [comment]
Looks Correct To Me.

**Pull request merged**